### PR TITLE
feat: clone preview without private spaces

### PR DESCRIPTION
**Pull request by rephus**
Closes: #9386

### Description:

Screenshot from 2024-03-20 11-07-46
Screenshot from 2024-03-20 11-08-32

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/services/ProjectService/ProjectService.ts [hunk 1]
```diff
@@ -410,7 +410,7 @@ export class ProjectService {
                 // We only allow copying from projects if the user is an admin until we remove the `createProjectAccess` call above
                 if (
                     user.ability.cannot(
-                        'manage',
+                        'create',
```
Comment (rephus) [inline packages/backend/src/services/ProjectService/ProjectService.ts:413]: interactive_viewer and above

**Review by ZeRego — APPROVED** [review]
lgtm 👍 

We need to update docs as well
Screenshot 2024-03-20 at 15 25 09

File: packages/backend/src/services/ProjectService/ProjectService.ts (hunk 1 above)
Comment (ZeRego) [reply packages/backend/src/services/ProjectService/ProjectService.ts:413]: I would have set this to `view` as this check is to check if the user has access to this project. The check for creating a project was before the action to create a project.

Comment (rephus) [reply packages/backend/src/services/ProjectService/ProjectService.ts:413]: perhaps we can remove the ability  check here, you're right, the `create` was done already.

**Pull request merged**